sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderClient.java

```diff
@@ -85,18 +93,23 @@
      * @throws ErrorResponseException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return the response.
+     * @throws IOException
      */
+
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public InputStream getMapTileV2(
-            TilesetID tilesetId,
-            TileIndex tileIndex,
-            OffsetDateTime timeStamp,
-            MapTileSize tileSize,
-            String language,
-            LocalizedMapView localizedMapView) {
-        return this.serviceClient.getMapTileV2(tilesetId, tileIndex, timeStamp, tileSize, language, localizedMapView);
+    public InputStream getMapTile(MapTileOptions options) throws IOException {
+        Mono<byte[]> byteArray = FluxUtil.collectBytesInByteBufferStream(this.asyncClient.getMapTile(options));
```

Comment:
As we discussed, we need to get the InputStream from StreamResponse, not from a byte array. Here we have already loaded up the bytes in memory, we don't need an input stream around it